MerlinAU.sh

```diff
@@ -3700,8 +3700,12 @@ Please manually update to version $minimum_supported_version or higher to use th
         release_link="$2"
     else
         Say "${REDct}**ERROR**${NOct}: No firmware release URL was found for [$PRODUCT_ID] router model."
-        "$inMenuMode" && _WaitForEnterKey_ "$mainMenuReturnPromptStr"
-        return 1
+        if [ "$inMenuMode" = true ]; then
+            _WaitForEnterKey_ "$mainMenuReturnPromptStr"
+            return 1
+        else 
+            _DoExit_ 1
```

Comment:
I don't see the need to call "`_DoExit_ 1`" here. When in "Menu Mode" we return to the Main Menu; otherwise, there's already a call to "`_DoExit_ 0`" to gracefully terminate upon returning.

Reply:
@Martinski4GitHub 

Adjusted in commit: https://github.com/ExtremeFiretop/MerlinAutoUpdate-Router/pull/207/commits/3c3453084a41a7557ae3229b137b3398a9b39b69